notes/avs.vbs

```diff
@@ -44,6 +44,7 @@ changelog = array()
 
 'INSERT ACTUAL CHANGES HERE, WITH PARAMETERS DATE, DESCRIPTION, AND SCRIPTWRITER. **ENSURE THE MOST RECENT CHANGE GOES ON TOP!!**
 'Example: call changelog_update("01/01/2000", "The script has been updated to fix a typo on the initial dialog.", "Jane Public, Oak County")
+call changelog_update("05/01/2023", "Updated TIKL to be set for 11 days instead of 10 to align with timeline in ONEsource manual for AVS.", "Mark Riegel, Hennepin County")
```

Comment:
Clarify that this 11 day TIKL is for the AVS Portal Case Review reminder after submitting an Ad-Hoc request.

Reply:
Updated changelog language to clarify